int32 ownerId
Rectangle[] reservations
float64 bid